src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+import core.basesyntax.impl.CsvFileReaderImpl;
+import core.basesyntax.impl.CsvFileWriterImpl;
+import core.basesyntax.impl.DataParserImpl;
+import core.basesyntax.impl.OperatorStrategyImpl;
+import core.basesyntax.impl.ProductCalculatorImpl;
+import core.basesyntax.impl.ReportCreatorImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.operators.BalanceOperator;
+import core.basesyntax.operators.Operator;
+import core.basesyntax.operators.PurchaseOperator;
+import core.basesyntax.operators.ReturnOperator;
+import core.basesyntax.operators.SupplyOperator;
+import core.basesyntax.service.DataParser;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.FileWriter;
+import core.basesyntax.service.ProductCalculator;
+import core.basesyntax.service.ReportCreator;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String inputFileName = "src/inputFile.scv";
+    private static final String outputFileName = "src/reportFile.csv";

```

Comment:
Its better to create a new package that include all files. 
src/main/resources/files..
Like in "Common mistakes".